docs/sampling.asciidoc

```diff
@@ -1,101 +1,129 @@
+A downside of tail-based sampling is that it results in more data being sent from APM agents to the APM Server.
+The APM Server will therefore use more CPU, memory, and disk than with head-based sampling.
+However, because the tail-based sampling decision happens in APM Server, there is less data to transfer from APM Server to {es}.
+So running APM Server close to your instrumented services can reduce any increase in transfer costs that tail-based sampling brings.
 
-* Transaction duration and transactions per minute
-* Transaction breakdown metrics
-* Errors, error occurrence, and error rate
+See <<configure-tail-based-sampling>> to get started.
 
-// To turn off the sending of all data, including transaction and error data, set `active` to `false`.
+**Distributed tracing with tail-based sampling**
 
-[float]
-==== Sample rates
+With tail-based sampling, all traces are observed and a sampling decision is only made once a trace completes.
 
-What's the best sampling rate? Unfortunately, there isn't one.
-Sampling is dependent on your data, the throughput of your application, data retainment policies, and other factors.
-A sampling rate from `.1%` to `100%` would all be considered normal.
-You may even decide to have a unique sample rate per service -- for example, if a certain service
-experiences considerably more or less traffic than another.
+In this example, `Service A` initiates four transactions.
+If our sample rate is `.5` (`50%`) for traces with a `success` outcome,
+and `1` (`100%`) for traces with a `failure` outcome,
+the sampled traces would look something like this:
 
-// Regardless, cost conscious customers are likely to be fine with a lower sample rate.
+image::./images/dt-sampling-example-3.png[Distributed tracing and tail based sampling example one]
 
 [float]
-==== Sampling with distributed tracing
+=== Sampled data and visualizations
 
-The initiating service makes the sampling decision in a distributed trace,
-and all downstream services respect that decision.
+A sampled trace retains all data associated with it.
+A non-sampled trace drops all <<data-model-spans,span>> and <<data-model-transactions,transaction>> data^1^.
+Regardless of the sampling decision, all traces retain <<data-model-errors,error>> data.
 
-In each example below, `Service A` initiates four transactions.
-In the first example, `Service A` samples at `.5` (`50%`). In the second, `Service A` samples at `1` (`100%`).
-Each subsequent service respects the initial sampling decision, regardless of their configured sample rate.
-The result is a sampling percentage that matches the initiating service:
+Sampling does not impact aggregated transaction and span <<data-model-metrics,metrics>>.
+This means that the visualizations in the APM app, like latency, throughput, time spent by span type, failed transaction rate,
+errors, error rate, etc., will always accurately reflect *all* of your application's requests,
+regardless of the configured sampling rate.
```

Comment:
This might be a bit of an overstatement. Sampling may affect accuracy of latency.

Metrics are based on the sampled traces, and weighted by the inverse sampling rate. e.g. if you are sampling at 5%, then each trace will be counted as 20. Thus, as variance of latency increases, or the sampling rate decreases, your level of error will increase.

Reply:
Hmm. Updated. I'm still a bit confused about this so please let me know if it still needs work (likely).